pytest_factoryboy/fixture.py

```diff
@@ -197,7 +196,7 @@ def inject_into_caller(name: str, function: Callable[..., Any], locals_: dict[st
     locals_[name] = function
 
 
-def get_model_name(factory_class: FactoryType) -> str:
+def get_model_name(factory_class: F) -> str:
```

Comment:
this can stay as it was before, as the type var does not appear anywhere else in the function signature